### PR TITLE
fix: change grpc and sentinel version

### PR DESCRIPTION
**Pull request by LaurenceLiZhixin**
change grpc versin to 1.26.0
change sentinal version to 0.6.2

### CODE REVIEW

**Review by zouyx — COMMENTED** [review]
Checking not pass

File: go.mod [hunk 1]
```diff
@@ -15,8 +17,9 @@ require (
...
 	github.com/uber/jaeger-client-go v2.22.1+incompatible
 	github.com/uber/jaeger-lib v2.2.0+incompatible // indirect
-	google.golang.org/grpc v1.27.0
-	google.golang.org/protobuf v1.25.0 // indirect
+	google.golang.org/grpc v1.26.0
 )
 
 go 1.13
+
+replace github.com/apache/dubbo-go v1.5.2-0.20201205063214-68ff8be79d6d => ../../../dubbo-go
```
Comment (AlexStocks) [inline go.mod:25]: r u kidding me?

Comment (LaurenceLiZhixin) [reply go.mod]: fixed

**Comment by LaurenceLiZhixin** [comment]
fixed cicd!

**Review by AlexStocks — APPROVED** [review]

**Review by zouyx — APPROVED** [review]

**Review by cityiron — APPROVED** [review]

**Pull request merged**